rosdep/python.yaml

```diff
@@ -5809,6 +5817,19 @@ python3-tornado:
       packages: [tornado]
   rhel: ['python%{python3_pkgversion}-tornado']
   ubuntu: [python3-tornado]
+python3-tqdm:
+  alpine: [py3-tqdm]
+  debian:
+    stretch: [python3-tqdm]
```

Comment:
This is available for all of our supported Debian versions (at this point I'm just ignoring Jessie), so:

```
debian: [python3-tqdm]
```

should suffice.